packages/core/src/stateUtils.ts

```diff
@@ -856,35 +812,8 @@ export function getActions<TContext, TEvent extends EventObject>(
   stateNode: StateNode<TContext, any, TEvent>,
   transition: StateTransition<TContext, TEvent>,
   currentContext: TContext,
-  _event: SCXML.Event<TEvent>,
-  prevState?: State<TContext>
+  _event: SCXML.Event<TEvent>
 ): Array<ActionObject<TContext, TEvent>> {
-  const prevConfig = getConfiguration(
```

Comment:
All of this code was moved, because it didn't belong in this function (mutated `transition`)

Reply:
A really nice change 👍